extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesCommonHelper.java

```diff
@@ -208,13 +207,11 @@ private static Collection<DecoratorBuildItem> createLabelDecorators(Optional<Pro
         });
 
         if (!config.isAddVersionToLabelSelectors()) {
-            result.add(new DecoratorBuildItem(target, new RemoveLabelDecorator(name, Labels.VERSION)));
             result.add(new DecoratorBuildItem(target, new RemoveFromSelectorDecorator(name, Labels.VERSION)));
             result.add(new DecoratorBuildItem(target, new RemoveFromMatchingLabelsDecorator(name, Labels.VERSION)));
         }
 
         if (!config.isAddNameToLabelSelectors()) {
-            result.add(new DecoratorBuildItem(target, new RemoveLabelDecorator(name, Labels.NAME)));
```

Comment:
@geoand (working on the 2.13 backports with @rsvoboda) we saw that you are doing the same fix for the name but we don't see it mentioned anywhere. You positive it needs fixing too? If so could you adjust the title of the PR so that the change is properly documented in the changelog?

Thanks!

Reply:
I am yeah, it's the same exact logic

Reply:
PR title updated accordingly

Reply:
We tried to backport it to 2.13 but this method doesn't exist at all (and I don't see something similar in this class). So either we should backport the previous fix that introduced this method or we can skip it.

I will let you and @Sgitario decide what to do with it. Please ping me loudly on Zulip if an action on my side is needed.

Reply:
I don't think that disabling the selectors is a super important feature that we should backport to 2.13, on the other hand, the changes are not very complex. 
I would say to not backport the feature and hence these changes.